configure.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
No, there's no bash on FreeBSD by default, and making a script that doesn't need it depend on it is a bad practice.